Makefile

```diff
@@ -330,15 +331,11 @@ else
 endif
 
 ifneq ($(filter y,$(DEP_ZLIB)),)
+	EXE_LIBS += $(MINIZ_OBJS) $(COMPRESSION_OBJS)
 	CXXFLAGS += -DDEPENDENCY_ZLIB
-	ifeq ($(OS),osx)
-		CXXLIBS += "-lz"
-	else
-		CXXLIBS += "-l:libz.a"
-	endif
 	QBLIB_NAME := $(addsuffix 1,$(QBLIB_NAME))
```

Comment:
I really like these changes, since you removed usage of `DEPENDENCY_ZLIB` from `libqb.cpp` and `qbx.cpp` we can actually go a bit further and remove the `CXXFLAGS` addition and also the `QBLIB_NAME` addition.

Reply:
Ah. Did not think of that. I'll get those removed.